### ISSUE TITLE
test_openmp setup error on the installed package

### ISSUE THREAD

**Issue by olebole** [astropy/extension-helpers#17]
When I install the package and then try to run the tests, I get the following error:
```
____________________________ ERROR at setup of test_add_openmp_flags_if_available _____________________________

self = <_pytest.config.Config object at 0x7f1f292ea760>, name = '--openmp-expected', default = <NOTSET>
skip = False

    def getoption(self, name, default=notset, skip=False):
        """…"""
        name = self._opt2dest.get(name, name)
        try:
>           val = getattr(self.option, name)
E           AttributeError: 'Namespace' object has no attribute '--openmp-expected'

/usr/lib/python3/dist-packages/_pytest/config/__init__.py:1054: AttributeError

During handling of the above exception, another exception occurred:

request = <SubRequest 'openmp_expected' for <Function test_add_openmp_flags_if_available>>

    @pytest.fixture
    def openmp_expected(request):
>       openmp_expected = request.config.getoption("--openmp-expected")

/usr/lib/python3/dist-packages/extension_helpers/tests/test_openmp_helpers.py:15: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <_pytest.config.Config object at 0x7f1f292ea760>, name = '--openmp-expected', default = <NOTSET>
skip = False

    def getoption(self, name, default=notset, skip=False):
        """…"""
        name = self._opt2dest.get(name, name)
        try:
            val = getattr(self.option, name)
            if val is None and skip:
                raise AttributeError(name)
            return val
        except AttributeError:
            if default is not notset:
                return default
            if skip:
                import pytest
    
                pytest.skip("no %r option found" % (name,))
>           raise ValueError("no option named %r" % (name,))
E           ValueError: no option named '--openmp-expected'

/usr/lib/python3/dist-packages/_pytest/config/__init__.py:1065: ValueError
______________________________ ERROR at setup of test_generate_openmp_enabled_py _______________________________

self = <_pytest.config.Config object at 0x7f1f292ea760>, name = '--openmp-expected', default = <NOTSET>
skip = False

    def getoption(self, name, default=notset, skip=False):
        """…"""
        name = self._opt2dest.get(name, name)
        try:
>           val = getattr(self.option, name)
E           AttributeError: 'Namespace' object has no attribute '--openmp-expected'

/usr/lib/python3/dist-packages/_pytest/config/__init__.py:1054: AttributeError

During handling of the above exception, another exception occurred:

request = <SubRequest 'openmp_expected' for <Function test_generate_openmp_enabled_py>>

    @pytest.fixture
    def openmp_expected(request):
>       openmp_expected = request.config.getoption("--openmp-expected")

/usr/lib/python3/dist-packages/extension_helpers/tests/test_openmp_helpers.py:15: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <_pytest.config.Config object at 0x7f1f292ea760>, name = '--openmp-expected', default = <NOTSET>
skip = False

    def getoption(self, name, default=notset, skip=False):
        """…"""
        name = self._opt2dest.get(name, name)
        try:
            val = getattr(self.option, name)
            if val is None and skip:
                raise AttributeError(name)
            return val
        except AttributeError:
            if default is not notset:
                return default
            if skip:
                import pytest
    
                pytest.skip("no %r option found" % (name,))
>           raise ValueError("no option named %r" % (name,))
E           ValueError: no option named '--openmp-expected'
```
Since the test runs nicely during the build, I would suspect that I miss something; however I have the same dependencies installed (namely the distutils, and setuptools packages, and the Python development environment).

**Comment by lupinix** [comment]
Getting the same while attempting to package for Fedora.

**Comment by astrofrog** [comment]
I believe this will be fixed by https://github.com/astropy/extension-helpers/pull/37

**Comment by astrofrog** [comment]
@olebole @lupinix - this should be fixed in extension-helpers 1.0.0, released today. If not, please feel free to re-open!
